table/tables/partition_test.go

```diff
@@ -299,7 +299,7 @@ func (ts *testSuite) TestTimeZoneChange(c *C) {
 	tk.MustExec("use test")
 	createTable := `CREATE TABLE timezone_test (
 	id int(11) NOT NULL,
-	creation_dt timestamp DEFAULT CURRENT_TIMESTAMP ) PARTITION BY RANGE ( unix_timestamp(creation_dt) )
+	creation_dt timestamp DEFAULT CURRENT_TIMESTAMP ) PARTITION BY RANGE ( ` + "UNIX_TIMESTAMP(`creation_dt`)" + ` )
```

Comment:
This function name changed to uppercase. Where did you change the code?

Reply:
@zimulala https://github.com/pingcap/tidb/pull/15806/commits/72d8a204033e5dad2b3b3844dae0fb639775a4f1#diff-07905132777c6fce0d33bf8f27a5dfaeL93-R97